Find a functionally equivalent code:android.os.Process.myPid:COMMENT
Method Modifier: public      static      final       
Comment:/**
 * Returns the identifier of this process, which can be used with
 * {@link #killProcess} and {@link #sendSignal}.
 */

@@ -1,3 +1,3 @@
 {
-    return Libcore.os.getpid();
+    return Os.getpid();
 }

